pos_barcode_sync/doc/index.rst

```diff
@@ -0,0 +1,8 @@
+=========================================
+ Barcode synchronization in POS
+=========================================
+
+Installation
+============
+
+* `Install <https://odoo-development.readthedocs.io/en/latest/odoo/usage/install-module.html>`__ this module in a usual way
```

Comment:
It's not the usual installation, actually. Longpolling has to be configured too.